stories/page-footer/page-footer.stories.js

```diff
@@ -0,0 +1,215 @@
+			<a class="fd-link fd-page-footer__row-item" id="fd-link-5" href="/" title="Participant Team">Participant Team</a>
+        </div>
+		<div class="fd-page-footer__text">
+			<span class="fd-form-label"> 1992-2021 @copyright all right reserved.</span>
+		</div>
+	</div>
+</div>
+`;
+
+singleLineLink.parameters = {
+    docs: {
+        iframeHeight: 100,
+        storyDescription:
+            'The Page Footer contains Image/logo, And list of links in the footer. It is responsive to tablet and mobile screen sizes.'
+    }
+};
+
+export const MultiLineLink = () => `
+<div class="fd-page-footer fd-page-footer--xl">
+	<div class="fd-page-footer__logo">	
+		<img src="//unpkg.com/fundamental-styles/dist/images/sap-logo.png" alt="page footer logo"></img>
+	</div>
+	<div class="fd-page-footer__container">
+		<div class="fd-page-footer__row">
+			<a class="fd-page-footer__row-item fd-link" id="fd-link-6" href="/" title="Supported Browser">Supported Browser</a>
+			<a class="fd-page-footer__row-item fd-link" id="fd-link-7" href="/" title="Security Disclosure">Security Disclosure</a>
+		</div>
+		<div class="fd-page-footer__row">
+			<a class="fd-page-footer__row-item fd-link" id="fd-link-8" href="/" title="Supported Browser">Supported Browser</a>
+			<a class="fd-page-footer__row-item fd-link" id="fd-link-9" href="/" title="Security Disclosure">Security Disclosure</a>
+			<a class="fd-page-footer__row-item fd-link" id="fd-link-10" href="/" title="Privacy Statement">Privacy Statement</a>
+			<a class="fd-page-footer__row-item fd-link" id="fd-link-11" href="/" title="Cookie Statement">Cookie Statement</a>
+			<a class="fd-page-footer__row-item fd-link" id="fd-link-12" href="/" title="Participant Team">Participant Team</a>
+		</div>
+		<div class="fd-page-footer__text">
+			<span class="fd-form-label"> 1992-2021 @copyright all right reserved.</span>
+		</div>
+	</div>
+</div>
+`;
+
+MultiLineLink.parameters = {
+    docs: {
+        iframeHeight: 100,
+        storyDescription:
+            'The Page Footer contains Image/logo, And list of links in the footer. To apply multiline link, add the <code class="docs-code">fd-page-footer__row</code> modifier class to the container element.'
+    }
+};
+
+export const ImageAndIconLink = () => `
+<div class="fd-page-footer fd-page-footer--xl">
+	<div class="fd-page-footer__logo">	
+		<img src="//unpkg.com/fundamental-styles/dist/images/sap-logo.png" alt="page footer logo"></img>
+	</div>
+	<div class="fd-page-footer__container">
+		<div class="fd-page-footer__row">
+			<a href="#" id="fd-link-13"  class="fd-page-footer__row-item fd-link" tabindex="0">
+			<span class="sap-icon--download sap-icon--s"></span>
+			 icon and link
+			</a>
+		</div>
+		<div class="fd-page-footer__row">
+			<a href="#" id="fd-link-14" class="fd-page-footer__row-item fd-link" tabindex="0">
+			<img class="fd-page-footer__row-item--image" src="https://picsum.photos/id/1021/50/50" alt="page footer clickable image link"/>
```

Comment:
What are you modifying with a modifier class if you don't have a base class?

You need the following format:
```
class="fd-block__element fd-block__element--modifier"
```

You can't have just:
```
class="fd-block__element--modifier"
```

A modifier class is used to change a property/properties of a base class